### ISSUE TITLE
Much higher performance of pretrained classifier on transferred sentences in Table1

### ISSUE THREAD

**Issue by AliceZhang2016** [shentianxiao/language-style-transfer#19]
Hi, 

For the sentiment dataset (Yelp), I used TextCNN (classifier.py) to train the classifier as mentioned in the original paper, and achieved accuracy = 97.54% (almost same as 97.4% in the paper). However, when I applied this pretrained classifier on the transferred sentences, it could achieve accuracy = 84.86, which is much higher than 78.4 in table1 of the paper, and accuracy=92.24 (>83.5 in table1) for the model Hu et al.

- **Train:** python classifier.py --train yelp/sentiment.train --dev yelp/sentiment.dev --output model_epoch_20/sentiment.dev --vocab yelp.vocab --model model_epoch_20/model --max_epochs 20
- **Test:**  python classifier.py --test sentiment.test.tsf.to --vocab yelp.vocab --model model_epoch_20/model --load_model true

where sentiment.test.tsf.to is the transferred sentences of the sentiment.test.

Could you please tell me how do you get the pretrained classifier's sentiment accuracy = 78.4 in table 1 ? Thank you very much.

**Comment by shentianxiao** [comment]
I changed the adversarial loss into the non-saturating one (i.e. min log (1-D(G(z))) -> max log D(G(z)), see the GAN paper). It becomes more stable and gets higher accuracy.

Also the implementation in https://github.com/asyml/texar/tree/master/examples/text_style_transfer is different from their original paper and improved over the code they previously provided us.

**Comment by AliceZhang2016** [comment]
ok, got it. Thanks !
